vm/src/obj/objstr.rs

```diff
@@ -1025,6 +1094,14 @@ impl PyString {
         let encoded = PyBytes::from_string(&self.value, &encoding, vm)?;
         Ok(encoded.into_pyobject(vm)?)
     }
+
+    #[pymethod(name = "__iter__")]
+    fn iter(zelf: PyRef<Self>, _vm: &VirtualMachine) -> PyStringIterator {
+        PyStringIterator {
+            position: Cell::new(0),
```

Comment:
I believe you're missing the `__reversed__`  method somewhere here?

Reply:
Actually, I checked it using CPython (3.7.3):

```
>>> hasattr(str, "__reversed__")
False
```

Should I implement ```__reversed__``` despite this fact or leave it unchanged?

Reply:
Thanks for your feedback :) I'll deliver corrections today.

Reply:
How is the `PyStringReverseIterator` used then?

Reply:
Within ```reverse```:

```
string = "123456789"
reversed_iter = reversed(string)
```

Reply:
Right, but nowhere in your code do you construct a `PyStringReverseIterator`, so there's no way to actually get one in Python code.

Reply:
Ok! I'll fix it in a moment.